network/test/epochtransition_test.go

```diff
@@ -127,6 +127,7 @@ func (t *testNodeList) networks() []*p2p.Network {
 }
 
 func TestEpochTransitionTestSuite(t *testing.T) {
+	t.Skip("flaky")
```

Comment:
[Getting this error on CI ](https://github.com/onflow/flow-go/pull/417/checks?check_run_id=1920863056)
```
{"level":"error","error":"could not generate topology: could not generate fanout for topic sync-cluster-flow-testnet: failed to find cluster peers for node e121b1728cdefec4de6b4218b2d2beb0ec6f0ee90e90a967e51bc7a9eb4a2ef3: failed to find the cluster for node ID e121b1728cdefec4de6b4218b2d2beb0ec6f0ee90e90a967e51bc7a9eb4a2ef3","message":"failed to update peers"}
--- FAIL: TestEpochTransitionTestSuite (11.67s)
    --- FAIL: TestEpochTransitionTestSuite/TestNodesAddedAndRemoved (7.05s)
        epochtransition_test.go:316: 
            	Error Trace:	epochtransition_test.go:316
            	            				epochtransition_test.go:290
            	Error:      	Condition never satisfied
            	Test:       	TestEpochTransitionTestSuite/TestNodesAddedAndRemoved
            	Messages:   	node is not connected to enough nodes
```

Reply:
Can we let the test to wait until the collection peers are ready? @yhassanzadeh13

Reply:
@zhangchiqing yes, but would be great to have @vishalchangrani confirmation on this as well.

Reply:
am ok with adding the skip now. @yhassanzadeh13 gave me some suggestions on how this test can be made more stable. I will work on it as part https://github.com/dapperlabs/flow-go/issues/5277